src/test/java/com/rultor/profiles/ProfilesTest.java

```diff
@@ -0,0 +1,115 @@
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.profiles;
+
+import com.jcabi.xml.XMLDocument;
+import com.rultor.spi.Profile;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+
+/**
+ * Tests for ${@link Profiles}.
+ * @author Maksimenko Vladimir (xupypr@xupypr.com)
+ * @version $Id$
+ * @since 1.62
+ */
+public class ProfilesTest {
+
+    /**
+     * Profiles can correctly merge master and fork profiles.
+     * @throws Exception In case of error.
+     */
+    @Ignore
+    public final void mergesProfiles() throws Exception {
+        final String architect = "Yegor512";
+        final String commander = "Yegor1024";
+        final String script = "do_another";
+        final String profile = StringUtils.join(
+            "<p><entry key='merge'>",
+            "<entry key='architect'><item>%s</item></entry>",
+            "<entry key='commander'><item>%s</item></entry>",
+            "<entry key='script'><item>%s</item></entry>",
+            "</entry></p>"
+        );
+        final Profile master = new Profile.Fixed(
+            new XMLDocument(
+                String.format(profile, architect, commander, "do_something")
+            )
+        );
+        final Profile fork = new Profile.Fixed(
+            new XMLDocument(
+                String.format(profile, "Bobby", "Sly Fox", script)
+            )
+        );
+        final Profile merged = new Profiles().merged(master, fork);
+        final List<String> architects = merged.read().xpath(
+            "//entry[@key='architect']/item/text()"
+        );
+        MatcherAssert.assertThat(
+            "There is one and only one architect",
+            architects.size(),
+            Matchers.is(1)
+        );
+        MatcherAssert.assertThat(
+            "Architect is taken from master",
+             architects.get(0),
+             Matchers.is(architect)
+        );
+        final List<String> commanders = merged.read().xpath(
+            "//entry[@key='commander']/item/text()"
+        );
+        MatcherAssert.assertThat(
+            "There is one and only one commander",
+            commanders.size(),
+            Matchers.is(1)
+        );
+        MatcherAssert.assertThat(
+            "Commander is taken from master",
+            commanders.get(0),
+            Matchers.is(commander)
+        );
+        final List<String> scripts = merged.read().xpath(
+            "//entry[@key='script']/item/text()"
+        );
+        MatcherAssert.assertThat(
+            "There is one and only one script",
+            scripts.size(),
+            Matchers.is(1)
+        );
+        MatcherAssert.assertThat(
+            "Script is taken from fork",
+            scripts.get(0),
+            Matchers.is(script)
```

Comment:
@xupyprmv there is `contains` matcher that verifies both size and contents at once